compiler/src/iree/compiler/Codegen/Common/PolynomialApproximationPass.cpp

```diff
@@ -29,24 +29,45 @@ namespace {
 class PolynomialApproximationPass final
     : public impl::PolynomialApproximationPassBase<
           PolynomialApproximationPass> {
+public:
+  using Base::Base;
+
   void runOnOperation() override {
+
+    using PatternFunction = llvm::function_ref<void(RewritePatternSet &)>;
+
+    auto populateErfPattern = [&](RewritePatternSet &patterns) {
+      if (clNativeMathPrecision) {
+        patterns.add<math::ErfPolynomialApproximation>(&getContext());
```

Comment:
nit:
```suggestion
    auto populateErfPattern = [](RewritePatternSet &patterns) {
      if (clNativeMathPrecision) {
        patterns.add<math::ErfPolynomialApproximation>(patterns.getContext());
```

I think this should work to make it a plain function?

Reply:
ahh, didn't know about this. Thanks, it gets much better.